package.json

```diff
@@ -76,6 +76,7 @@
   },
   "devDependencies": {
     "@types/react": "16.8.4",
+    "babel-plugin-styled-components": "^1.12.0",
```

Comment:
Currently we prefer strict versions in this project for dependencies, which affects bundle